src/test/resources/mappings/cached-opendistro-ism-config.json

```diff
@@ -443,7 +443,8 @@
             }
           }
         },
-        "ism_template": {
+        "ism_templates": {
```

Comment:
I think you have a few choices here.

What you current have done is a breaking change as there will be people in ODFE that are using "ism_template". We should try do avoid this.

Another option is to duplicate them, and in the parser you read in both and add them together in the data model.

Another option is to just not rename ism_template -> ism_templates.

And then there is an alias field type too, but that would need to be looked in to for how it behaves and if it could have any issues.

Reply:
still using ism_template, but now internally we treat it as a List, not a single object